homeassistant/components/roomba/irobot_base.py

```diff
@@ -194,6 +208,13 @@ def extra_state_attributes(self):
             state_attrs[ATTR_ERROR] = self.vacuum.error_message
             state_attrs[ATTR_ERROR_CODE] = self.vacuum.error_code
 
+        state_attrs[ATTR_BATTERY_CYCLES] = self._battery_stats.get(
+            "nLithChrg"
+        ) or self._battery_stats.get("nNimhChrg")
+
+        average_mission_time = self._mission_stats.get("aMssnM")
+        state_attrs[ATTR_AVERAGE_MISSION_TIME] = f"{average_mission_time} minutes"
```

Comment:
Not sure if there is a pattern to support translation but done this way this would be hard coded to English.

Reply:
I have my HA in german and all entities that have attributes I could not find a single one that has translated attribute names. They are all in english. So I don't think they are meant to be translated.

Reply:
We generally longer add new `extra_state_attributes`. These should be their own first class sensors with proper device classes.

Reply:
I guess you meant "no longer"?
So should I convert the attributes to sensors if I understand correctly?

Reply:
> I guess you meant "no longer"?
Yes, fixed

>  So should I convert the attributes to sensors if I understand correctly?
Yes please